addons/project/static/src/js/project_calendar.js

```diff
@@ -7,7 +7,7 @@ import { ProjectControlPanel } from '@project/js/project_control_panel';
 
 const ProjectCalendarController = CalendarController.extend({
     _renderButtonsParameters() {
-        return Object.assign({}, this._super(...arguments), {scaleDrop: true});
+        return Object.assign({}, this._super(...arguments));
```

Comment:
I think you can remove this override since we no longer add additional parameters. 🙂